lib/runner/extensions/event.command.js

```diff
@@ -183,33 +238,52 @@ module.exports = {
                     });
 
                 this.host.on(EXECUTION_REQUEST_EVENT_BASE + executionId, function (cursor, id, requestId, request) {
-                    // clone cursor and add source
-                    var cursorWithSource = _.clone(cursor),
+                    // remove files in request body if any
+                    sanitizeFiles(request, function (err, sanitizedRequest) {
+                        // @todo - there is no error flows now
+                        // this is here to allow extending sanitizeFiles
+                        // to include error flows future
+                        if (err) {
+                            return;
+                        }
+
+                        var tempRequest = request,
```

Comment:
we would not need `tempRequest` variable if sanitiseFIles always return a request or null if request itself is missing / cannot be constructed.